src/Parser/Header.php

```diff
@@ -47,6 +48,26 @@ public function __construct(string $content)
         $this->content = $content;
     }
 
+    public function harvestPaxData(string $paxData): void
+    {
+        foreach (explode(PHP_EOL, $paxData) as $record) {
```

Comment:
Be aware the `PHP_EOL` constant! It's so risky. On some platforms it can cause to unpredicted failures (e.g. on Windows contains `"\n\r"`), PAX format [expects `\n` only](https://pubs.opengroup.org/onlinepubs/9699919799/utilities/pax.html#tag_20_92_13_02:~:text=%22%25d%20%25s%3D%25s%5Cn%22%2C%20%3Clength%3E%2C%20%3Ckeyword%3E%2C%20%3Cvalue%3E).

Reply:
[bf20e35](https://github.com/jakubboucek/php-tar-stream-reader/pull/5/commits/bf20e3590e749eb602d3872448c1611f46e40fb8)